### ISSUE TITLE
<Tile> prop "activeOpacity " doesn't work

### ISSUE THREAD

**Issue by keith527** [react-native-training/react-native-elements#479]
### Description
I use the <Tile> in my Demo, But I want to set activeOpacity to "0.95",but doesn't work.

There is my code:

``` 
<Tile imageSrc={require('../image/sea.jpg')}
                 activeOpacity={0.95}
                 onPress={() => alert('onPress')}
            />
```

I use the latest version of react-native-elements

**Comment by Monte9** [comment]
Unable to reproduce it. Please see this [Snack demo](https://snack.expo.io/BJ0nMeGBZ)

**Comment by mateen-hussain** [comment]
@Monte9 I have the same problem too. I'm trying to disable the touch feedback on a tile.

```
<Tile
			imageSrc={defaultImagePath}
			title={registration}
			containerStyle={styles.TileContainer}
			titleStyle={styles.TileTitle}
			imageContainerStyle={styles.TileIconStyle}
			activeOpacity={1}
		/>
```

I tried various values between 0-1 to disable the touchfeedback but I dont see any effect.

```
    "react": "16.0.0",
    "react-native": "0.50.0",
    "react-native-elements": "0.17.0",
```

**Comment by xavier-villelegier** [comment]
@mateen-hussain Yes you're right.
You can see that the `activeOpacity` is taken [here](https://github.com/react-native-training/react-native-elements/blob/master/src/tile/Tile.js#L27) but not passed to the `TouchableOpacity`, because not in `...atributes` anymore.

Feel free to open a PR to fix it, it is as simple as adding `activeOpacity={activeOpacity}` to the `TouchableOpacity` 👌

**Comment by mateen-hussain** [comment]
Would love to contribute. let me give it a try.

**Comment by xavier-villelegier** [comment]
@mateen-hussain, @lex111 has already opened a PR, have a look at #839.
There is a bunch of "good first issue"s you can try to take down ! Give [them](https://github.com/react-native-training/react-native-elements/issues?q=is%3Aissue+is%3Aopen+label%3A%22good+first+issue%22) a shot 💪

**Comment by xavier-villelegier** [comment]
Hey guys, fixed in `v1.0.0-beta2`, just released 👍
